NIPTool/models/database/user.py

```diff
@@ -1,7 +1,13 @@
+from datetime import datetime
+from typing import Optional
+
 from pydantic import BaseModel
 
 
 class User(BaseModel):
-    email: str
     username: str
-    role: str
+    email: Optional[str] = None
+    disabled: Optional[bool] = None
+    added: Optional[datetime] = None
+    role: Optional[str] = None
+    hashed_password: Optional[str]
```

Comment:
Why are these optional? Looks to me like they should all be mandatory
```suggestion
    email: EmailStr
    disabled: bool = False
    added: datetime = datetime.now()
    role: Literal["admin", "user"]
    hashed_password: str
```
not sure about the roles

Reply:
If there is a need to have different user objects for different situations I think we should create them instead of setting everything to optional

Reply:
Good point :)